t/42-df-based-cleanup.t

```diff
@@ -164,7 +164,7 @@ subtest 'unable to make enough room; important job scheduled during the cleanup
     is $job->{result}, 'Unable to cleanup enough results', 'unable to make enough room';
 };
 
-my $new_job_id = $jobs->search({}, {order_by => {-desc => 'id'}})->first->id;
+my $new_job_id = $jobs->find({}, {order_by => {-desc => 'id'}})->id;
```

Comment:
`find` without any search attributes?

Reply:
This one should definitely be limited to 1 row (as the intend it to get only the "newest" job here).